src/components/scenes/RampCreateScene.tsx

```diff
@@ -246,12 +246,12 @@ export const RampCreateScene: React.FC<Props> = (props: Props) => {
     if (fiatUsdRate == null) return
     let abort = false
     const applyInitial = async (): Promise<void> => {
-      if (abort) return
+      if (abort || direction !== 'buy') return
```

Comment:
Why are we adding this constraint? Why not have this default amount for sell too?

Reply:
Punted task https://app.asana.com/1/9976422036640/project/1201386023359452/task/1211745454961037?focus=true